packages/shared/src/components/utilities.tsx

```diff
@@ -16,6 +16,12 @@ export enum Theme {
   Lettuce = 'lettuce',
 }
 
+export enum Justify {
+  End = 'justify-end',
+  Center = 'justify-center',
+  Between = 'justify-between',
+}
```

Comment:
Love this enum. We should probably do this as well on some other CSS properties that we tend to clash with sometimes. Probably when we face it on the road.

Reply:
Yep would be a great idea going forward, enums just make things more explicit and set in stone 🥳

Reply:
I love this. We should use it on more utility classes (if they are passed as props)